mantidproject/mantid
# Move TofConverter to qtpy and add to workbench

**To test:**
Start the TofConverter interface in both workbench and mantidplot and see that everything works

You should also be able to start it by itself.
`bin/mantidpython .../scripts/TofConverter.py`

*There is no associated issue.*


*This does not require release notes* because workbench


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
